libs/s25main/Settings.cpp

```diff
@@ -307,7 +315,11 @@ void Settings::Load()
         // interface
         // {
         interface.autosave_interval = iniInterface->getIntValue("autosave_interval");
-        interface.invertMouse = iniInterface->getValue("invert_mouse", false);
+#ifdef __ANDROID__
+        interface.mouseMode = iniInterface->getValue("mouse_mode", 2);
+#else
+        interface.mouseMode = iniInterface->getValue("mouse_mode", 0);
+#endif
```

Comment:
the 2 and 0 are even used twice (see above)

make a named constant and use it. (we should have done that anyway already)

basically we should first try to get  `mouse_mode` and if it does not exist try to read `invert_mouse` instead.